packages/smooth_app/lib/pages/product/edit_product_page.dart

```diff
@@ -146,6 +151,12 @@ class _EditProductPageState extends State<EditProductPage> {
                 subtitle:
                     appLocalizations.edit_product_form_item_photos_subtitle,
                 onTap: () async {
+                  if (!await ProductRefresher().checkIfLoggedIn(context)) {
+                    return;
+                  }
```

Comment:
Why did you add this, AFAIK you should be able to upload pictures without being logged in

Reply:
this was something from earlier on, not something i did in this pr, though if we need to remove it then we can get rid of it for sure , for now i don't have any opinions

Reply:
@AshAman999 From #3680 #3689 we do not check the "logged-in" status here.
I guess you haven't refreshed your local code with the latest merges.
Please remove that test :)

Reply:
I was replying on my git lens extension, my bad, removed it now